src/test/java/Pages/HomePage.java

```diff
@@ -0,0 +1,67 @@
+package Pages;
+
+import org.openqa.selenium.By;
+import org.openqa.selenium.JavascriptExecutor;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.interactions.Actions;
+import org.openqa.selenium.support.ui.ExpectedConditions;
+import org.testng.Assert;
+
+public class HomePage extends BasePage{
+
+    public HomePage(WebDriver givenDriver) {
+        super(givenDriver);
+    }
+    By avatar = By.cssSelector(".avatar");
+    By playBtnFooter = By.cssSelector("[data-testid='play-btn']");
+    By pauseBtnFooter = By.cssSelector("[data-testid='pause-btn']");
+    By searchField = By.cssSelector("[type='search']");
+    By viewAllButton = By.cssSelector("button[data-test='view-all-songs-btn']");
+    By firstSongSelect = By.cssSelector("[class='song-list-wrap main-scroll-wrap search-results'] .items");
+    By addToButton = By.cssSelector(".btn-add-to");
+    By playlistNameField = By.cssSelector("[id='songResultsWrapper'] [placeholder='Playlist name']");
```

Comment:
You should use encapsulation of non-static and non-final fields.

If a variable is used only by other methods of that class, it should be declared as private.

I recommend:  make all the locators and WebElements private, because private keyword in java allows most restrictive access to variables and methods and offer strongest form of Encapsulation. 

For example

    ```
    private By shuffleBtnLocator = By.cssSelector(".btn-shuffle-all");
    private By firstSongLocator = By.cssSelector(".play");

```